api/values_service.py

```diff
@@ -19,7 +19,7 @@ def getCurrentPrice(currency=None):
   if input[:2].upper() == "SP":
     protocol1='Bitcoin'
     pid1=getPropertyid('BTC', protocol1)
-    protocol2='Mastercoin'
+    protocol2='Omni'
```

Comment:
@Nevtep   protocol2 needs to remain Mastercoin on these changes  for a little while longer until i can finish teh db updates